### ISSUE TITLE
Is `skip_spatial` the result of `F_L` ?

### ISSUE THREAD

**Issue by cnjanus** [liruihui/Dis-PU#8]
I have a question about the local refinement unit. 
In the essay, the author writed 'Next, to obtain the local point feature overFL, a common routine is to apply MLPs followed by a max-pooling along theK-dimension.' But what I find in code is that 
`skip_spatial = tf.reduce_max(grouped_feat, axis=[2])
 skip_spatial = tf_util.conv1d(skip_spatial, mlp[-1], 1, padding='VALID', stride=1,bn=use_bn, is_training=is_training, scope='skip',bn_decay=bn_decay, weight_decay=weight_decay)`  
and `grouped_feat = tf.add(grouped_feat, skip_spatial)`.
So I want to determine wheter `skip_spatial` is the result of `F_L` ?

**Comment by anshen666** [comment]
> I have a question about the local refinement unit. In the essay, the author writed 'Next, to obtain the local point feature overFL, a common routine is to apply MLPs followed by a max-pooling along theK-dimension.' But what I find in code is that `skip_spatial = tf.reduce_max(grouped_feat, axis=[2]) skip_spatial = tf_util.conv1d(skip_spatial, mlp[-1], 1, padding='VALID', stride=1,bn=use_bn, is_training=is_training, scope='skip',bn_decay=bn_decay, weight_decay=weight_decay)` and `grouped_feat = tf.add(grouped_feat, skip_spatial)`. So I want to determine wheter `skip_spatial` is the result of `F_L` ?

请问方便留个联系方式吗？这个项目一直没跑通，请问一定要在ubuntu环境下才可以跑通吗？
